src/main/java/tech/ypsilon/bbbot/discord/listener/ChannelListener.java

```diff
@@ -10,11 +10,16 @@
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
 import org.jetbrains.annotations.NotNull;
+import tech.ypsilon.bbbot.ButterBrot;
 import tech.ypsilon.bbbot.database.codecs.StudyGroupCodec;
 
-public class ChannelListener extends ListenerAdapter {
+public class ChannelListener extends ButterbrotListener {
     private final static long CAT_ID = 762052348259467275L;
```

Comment:
Hence you are into it anyway, could you pleas port this constant into the config?

Reply:
see c48b130
I have also put the hard-coded study group join channel into the configuration